src/measure_theory/measure/content.lean

```diff
@@ -362,6 +364,56 @@ begin
   exact (μ.le_outer_measure_compacts K).trans (le_to_measure_apply _ _ _)
 end
 
+end outer_measure
+
+section regular_contents
+
+/-- A content `μ` is called regular if for every compact set `K`,
+  `μ(K) = inf {μ(K') : K ⊂ int K' ⊂ K'`. See Paul Halmos (1950), Measure Theory, §54-/
+def content_regular := ∀ ⦃K : topological_space.compacts G⦄,
+  μ K = ⨅ (K' : topological_space.compacts G) (hK: (K : set G) ⊆ interior (K' : set G) ), μ K'
+
+lemma content_regular_exists_compact (H : content_regular μ) (K : topological_space.compacts G)
+  {ε : nnreal} (hε : ε ≠ 0) :
+  ∃ (K' : topological_space.compacts G), (K.carrier ⊆ interior K'.carrier) ∧ μ K' ≤ μ K + ε :=
```

Comment:
```suggestion
  ∃ (K' : topological_space.compacts G), ((K : set G) ⊆ interior (K' : set G)) ∧ μ K' ≤ μ K + ε :=
```

Reply:
Also, a more convenient statement would be to say that for all `x > μ K` we get a `K'` with `μ K' ≤ x`. Then the proof is just [`infi_lt_iff`](https://leanprover-community.github.io/mathlib_docs/order/complete_lattice.html#infi_lt_iff) and in the application you should be able to apply it if you use [`le_of_forall_lt`](https://leanprover-community.github.io/mathlib_docs/order/basic.html#le_of_forall_lt) instead of `ennreal.le_of_forall_pos_le_add`

Reply:
Would it not be worth it to keep the more specific existence statement, since it is so similar to `inner_content_exists_compact` ?

Reply:
I think those lemma statements should also be simplified. But if you prefer to keep it this way, that's also fine by me.

Reply:
In that case I prefer the statements as they are